influxdb/020-deploy-pods-service.md

```diff
@@ -0,0 +1,17 @@
+The next step is to create an InfluxDB host pod, a service that exposes port
+8086 on that pod, and a client pod for interaction with the host.
+
+Create the service account, service, StatefulSet and client pod from their
+respective yaml files:
+
+`kubectl create -f ./`{{execute}}
```

Comment:
When building an scenario we have to keep in mind the maintainability of the solution. We don't want to have to make the same change in 2 different places. In this case, if the use case needs changing then we would have to apply the changes into the scenario. That is not ideal. 

It is better to do the 
```
git clone ..../use-cases
cd storageos-usecases
```

```
kubectl create -f ./influxdb
```

In this way, people can also related to use the same yamls in their clusters if they want to.